### DISCUSSION TITLE
How to customize color of cursor cell

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#2017] [Q&A] [answered]
The cursor row has a different color than normal rows, which can be customized as `color_current_row`. However, I would like to customize the color of the cursor cell inside the cursor row. By default it's a brighter light gray than the light gray of the cursor row. I can't seem to find the option to change that. Is it customizable?

**Comment by anjakefala** [comment, marked as answer]
There is a `color_current_cell`! =) https://github.com/saulpw/visidata/blob/0ba5a116a8fedd67612cde64158a63d2cc33394e/visidata/interface.py#L43

**Reply by cool-RR** [reply]
Good, but why am I not seeing it? Was it not released in 2.11.1?

**Reply by anjakefala** [reply]
It was not! It used to be part of vdplus. I moved it into VisiData core for 3.0.

**Reply by cool-RR** [reply]
Great. I'll follow up for the next version. Thank you!

**Reply by cool-RR** [reply]
I tried `color_current_cell` now. It doesn't seem to have any effect. I set it to `red on yellow` to be extra atrocious, but the current cell still shows as a slightly more bright white than the current row.
